### PR TITLE
Refactor/singular service tests name

### PR DESCRIPTION
**Pull request by chedieck**
Description:
Continuation of #251, in which I forgot to change the name of the test files.

Test plan:
Nothing should change.

Depends on:
- [ ] #255

### CODE REVIEW

File: constants/index.ts [hunk 1]
```diff
@@ -9,9 +9,13 @@ export const RESPONSE_MESSAGES = {
   NAME_NOT_PROVIDED_400: { statusCode: 400, message: "'name' not provided." },
   NAME_ALREADY_EXISTS_400: { statusCode: 400, message: 'Button name already exists for this user.' },
   ADDRESSES_NOT_PROVIDED_400: { statusCode: 400, message: "'addresses' not provided." },
+  BUTTON_IDS_NOT_PROVIDED_400: { statusCode: 400, message: "'paybuttonIdList' not provided." },
   TRANSACTION_ID_NOT_PROVIDED_400: { statusCode: 400, message: "'transactionId' not provided." },
   INVALID_NETWORK_SLUG_400: { statusCode: 400, message: 'Invalid network slug.' },
   INVALID_BUTTON_DATA_400: { statusCode: 400, message: "'buttonData' is not valid JSON." },
+  PAYBUTTON_ALREADY_BELONGS_TO_WALLET_400: { statusCode: 400, message: 'Some of the paybuttons already belong to another wallet.' },
```
Comment (Klakurka) [inline constants/index.ts:16]: `One or more buttons already belong to another wallet.`

File: constants/index.ts [hunk 2]
```diff
@@ -9,9 +9,13 @@ export const RESPONSE_MESSAGES = {
...
+  WALLET_CREATION_FAILED_400: { statusCode: 400, message: 'Wallet creation failed.' },
+  ADDRESS_ALREADY_BELONGS_TO_WALLET_400: { statusCode: 400, message: "Some of the paybuttons' addresses already belong to another wallet." },
```
Comment (Klakurka) [inline constants/index.ts:18]: `One or more button addresses already belong to another wallet.`

**Review by Klakurka — CHANGES_REQUESTED** [review]

File: constants/index.ts (hunk 1 above)
Comment (chedieck) [reply constants/index.ts]: This is refering to #255 changes; addressed.

**Review by Klakurka — APPROVED** [review]

**Review by Klakurka — CHANGES_REQUESTED** [review]
Since we'll probably be supporting alt buttons in the future, can we have all of these constants use just the word 'button' instead of 'paybutton'?

**Review by Klakurka — APPROVED** [review]

**Pull request merged**